index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Add a `lang` attribute to the `html` tag to declare the language of this document.